packages/composites/circuit-breaker/src/adapter.ts

```diff
@@ -0,0 +1,65 @@
+import { AdapterError, Requester, util, Validator, Logger } from '@chainlink/ea-bootstrap'
+import { AdapterRequest, ExecuteWithConfig, Config, ExecuteFactory } from '@chainlink/types'
+import { makeConfig } from './config'
+import { AxiosResponse } from 'axios'
+
+const customParams = {
+  primarySource: true,
+  secondarySource: true,
+}
+
+export interface ResponseSchema {
```

Comment:
We should not include the definition here

Reply:
removed